controllers/app.controller.js

```diff
@@ -32,17 +34,40 @@ exports.getArticlesById = (req, res, next) => {
     .catch(next);
 };
 
-exports.getArticlesComments = (req, res, next) => {
+exports.getArticleComments = (req, res, next) => {
   const { article_id } = req.params;
-  const promises = [fetchArticlesComments(article_id)];
+  fetchArticlesById(article_id)
```

Comment:
I would consider refactoring to make use of a checkExists function that would be reusable as you'll need to do this check more than once.